docker/testing.entrypoint.sh

```diff
@@ -136,7 +136,7 @@ fi
 
 # Check results and exit accordingly.
 if [ -f "${TESTS_OUTPUT}/failed" ]; then
-    echo "Uh oh, some went wrong."
+    echo "Uh oh, something went wrong."
```

Comment:
This makes me smile. . .when there's something wrong with the error message stating that there's something wrong 🤣